tools/dataset_converters/nuscenes_converter.py

```diff
@@ -12,7 +12,7 @@
 from pyquaternion import Quaternion
 from shapely.geometry import MultiPoint, box
 
-from mmdet3d.datasets import NuScenesDataset
+from mmdet3d.datasets.convert_utils import NameMapping
```

Comment:
May consider renaming the NameMapping to NuScenesNameMapping because the convert_utils is not only used for nuScenes